docs-2.0/3.ngql-guide/11.edge-statements/3.upsert-edge.md

```diff
@@ -0,0 +1,42 @@
+# UPSERT EDGE
+
+```ngql
+UPSERT EDGE <src_vid> -> <dest_vid> [@rank] OF <edge_type> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
+```
+
+- `update_columns` is the properties of the edge to be updated. For example, `e1.col1 = $^.e1.col2 + 1` means to update `e1.col1` to `e1.col2+1`.
+
+    > **NOTE:**  `$^` indicates the edge to be updated.
+
+- `condition` is some constraints. Only when met, `UPSERT` is executed successfully. `condition` supports expression operations.
+- `columns` is the columns to be returned, `YIELD` returns the latest updated values.
+
+`UPSERT` is a combination of `UPDATE` and `INSERT`.  Use `UPSERT EDGE` to update properties on an edge if it exists or insert a new edge if it does not exist. The `UPDATE EDGE` statement only updates one edge type of an edge at a time.
```

Comment:
.  Use [ extra space]

Reply:
Only one edge type can be updated or inserted at a time with the `UPSERT EDGE` statement.